src/object/srv_obj.c

```diff
@@ -2270,6 +2270,14 @@ obj_ioc_init_oca(struct obj_io_context *ioc, daos_obj_id_t oid)
 static int
 obj_inflight_io_check(struct ds_cont_child *child, uint32_t opc, uint32_t flags)
 {
+	if (opc == DAOS_OBJ_RPC_ENUMERATE && flags & ORF_FOR_MIGRATION) {
+		if (child->sc_ec_agg_active) {
+			D_ERROR(DF_UUID" ec aggreate still active\n",
+				DP_UUID(child->sc_pool->spc_uuid));
+			return -DER_UPDATE_AGAIN;
```

Comment:
the err code UPDATE_AGAIN seems a bit confuse as it is for obj list operation.

Reply:
hmm, just want to reuse the ERROR code a bit. though it is a confuse indeed. probably need redefine